src/util/vfs/vfs-zip.c

```diff
@@ -597,10 +597,12 @@ struct VFile* _vdzOpenFile(struct VDir* vd, const char* path, int mode) {
 	UNUSED(mode);
 	struct VDirZip* vdz = (struct VDirZip*) vd;
 
+#ifndef _MSC_VER
 	if ((mode & O_ACCMODE) != O_RDONLY) {
```

Comment:
Yeah, looks like KDE's MSVC wrappers have something mostly identical to that. Where should I put that define? Along with the other MSVC defines in common.h?